docs/conceptual-guides/add-on-loader.md

```diff
@@ -0,0 +1,67 @@
+---
+myst:
+  html_meta:
+    "description": "An explanation of the add-ons configuration loader in @plone/registry"
+    "property=og:description": "An explanation of the add-ons configuration loader in @plone/registry"
+    "property=og:title": "Add-ons configuration loader"
+    "keywords": "Seven, @plone/registry, registry, add-ons, loader"
+---
+
+# Add-ons configuration loader
+
+Add-ons that are compatible with `@plone/registry` can load configuration into the configuration registry.
```

Comment:
What does it mean to be "compatible with `@plone/registry`"? This would be good to explain more.

Reply:
re-worded